### ISSUE TITLE
Visual Bugs that makes the game unplayable.

### ISSUE THREAD

**Issue by JoaoPedro159** [MyM-ModpackTeam/GalacticScience#209]
Hi, I was trying to play again this modpack and this happened:
![image](https://github.com/MyM-ModpackTeam/GalacticScience/assets/75337624/241711d2-af9f-4f81-88a7-943f75d44c6a)
![image](https://github.com/MyM-ModpackTeam/GalacticScience/assets/75337624/9063d2e5-d1d4-4395-ada2-c7583a21ae37)
![image](https://github.com/MyM-ModpackTeam/GalacticScience/assets/75337624/fee37e00-c5a9-4f5c-8b35-b61623ab8cbb)
Principally when I tried to hold any itens. I was playing in the default map and noticed that the observatory map happened less bugs than the default but some bugs still in the both maps.
I tried to download the optifine, change the graphics settings of the modpack, instance more memory, press F3+A to reload the chunks in the game but didn't resolve.
I even recorded a video: https://drive.google.com/file/d/1AhCKyDgifBjL1gW7x65rhCc0_lF9Y9go/view?usp=drive_link
So I'm here asking for help and I can give more details if someone know how to resolve this, please.

**Comment by Aiidoneus** [comment]
What version of the modpack are you playing on? 
What launcher are you using?

**Comment by JoaoPedro159** [comment]
Hello there, 
modpack version: "1.7.0-1.710"
Launcher: I'm using the FTB App

**Comment by JoaoPedro159** [comment]
Sorry missclick

**Comment by Aiidoneus** [comment]
> Hello there, 
> modpack version: "1.7.0-1.710"
> Launcher: I'm using the FTB App

Let me see if I can reproduce this. I didn't get this during my tests when I was updating the pack. I will get back in the next day or two

**Comment by JoaoPedro159** [comment]
HELLO EVERYONE! and I have good news: I fixed the visual Glitches!!!
Resolution: I Noticed that others FTB modpacks was having the same bugs for me.
I have a Nvidia and this video helped me to fix the glitches in any modpack: https://www.youtube.com/watch?v=N-EH9GnD5rA.
Observation: In the time 1:27 of the video, select the program "OpenJDK Plataform binary".
I hope that I helped someone, have a great day!!!
